_episodes/04-rdm.md

```diff
@@ -59,35 +65,99 @@ A Data Management Plan is a roadmap for you to manage your data efficiently and
+-   A distinction can be described in the plan separately for different data types such as raw (primary), processed and ready to use (finalised to publish) datasets.
+-   All types of data will have to be described to be placed into context by using metadata and adequate documentation which will allow anyone in your team to interpret the data in the future.
+3.  **Data storage and backup procedures** should be assessed for each project and established depending on the institutional requirements, associated costs and recommended format from your field. We will diuscuss this in detail later in this lesson.
+4.  **Preservation of the research outputs** can be managed differently based on if they can be made publicly available or not. Personal data or research outputs needed to apply for patents cannot be publicly shared but they still have to be preserved for several years, depending on the policies of your country, institute and funder. Learn more about this in detail in [Sharing and Archiving Data](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-sharing.html#rr-rdm-sharing) chapter in _The Turing Way_.
+5.  **Reuse of your research outputs** should be ensured by selecting licenses for different components of your research when you make your output available on a repository (see the Licensing subchapters on [data](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-data.html#rr-licensing-data) and [software](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-software.html#rr-licensing-software) for more information). A dedicated document (such as a README file) is recommended for describing research outputs into context. UK Data Service has provided a [Data Management Checklist](https://ukdataservice.ac.uk/learning-hub/research-data-management/plan-to-share/checklist/) to help cover different aspects of the DMP.
+
 
 ## The FAIR Principles
 
-Details TBA
+The FAIR guiding principles for scientific data management and stewardship are guidelines to improve the **F**indability, **A**ccessibility, **I**nteroperability and **R**eusability of digital assets; all of which support research reproducibility. The FAIR principles help create DMP for your project ensuring that research data is available to others for reuse.
+
+FAIR data should be:
+- **Findable**: The first step in (re)using data is to find them! Descriptive metadata (information about the data such as keywords) are essential.
+- **Accessible**: Once the user finds the data and software they need to know how to access it. Data could be openly available but it is also possible that authentication and authorisation procedures are necessary.
+- **Interoperable**: Data needs to be integrated with other data and interoperate with applications or workflows.
+- **Reusable**: Data should be well-described so that they can be used, combined, and extended in different settings.
```

Comment:
```suggestion
> ## FAIR data should be:
>
> - **Findable**: The first step in (re)using data is to find them! Descriptive metadata (information about the data such as keywords) are essential.
> - **Accessible**: Once the user finds the data and software they need to know how to access it. Data could be openly available but it is also possible that authentication and authorisation procedures are necessary.
> - **Interoperable**: Data needs to be integrated with other data and interoperate with applications or workflows.
> - **Reusable**: Data should be well-described so that they can be used, combined, and extended in different settings.
{: .callout}
```